src/build.rs

```diff
@@ -71,15 +78,16 @@ pub fn release(p: &crate::project::Project, version: &String) -> Result<(), Erro
     for file in &p.files {
         fs::copy(file, format!("releases/{}/@{}/{}", version, p.prefix, file))?;
     }
-    if !Path::new("keys").exists() {
-        fs::create_dir("keys")?;
+    if !Path::new("releases/keys").exists() {
+        fs::create_dir("releases/keys")?;
     }
-    if !Path::new(&format!("keys/{}.bikey", p.prefix)).exists() {
+    if !Path::new(&format!("releases/keys/{}.bikey", p.prefix)).exists() {
+        println!(" {} {}.bikey", "Generating".green().bold(), p.prefix);
         armake2::sign::cmd_keygen(PathBuf::from(&p.prefix))?;
-        fs::rename(format!("{}.bikey", p.prefix), format!("keys/{}.bikey", p.prefix))?;
-        fs::rename(format!("{}.biprivatekey", p.prefix), format!("keys/{}.biprivatekey", p.prefix))?;
```

Comment:
Should the private key be moved into releases? I thought they could be used for multiple releases. This way you could keep the same public key on servers so you know an update came from the same source as your last version.

Reply:
I think it's better to have them in releases folder (next to individual, yeah), for easy cleanup and isolated from other things.

Reply:
Yeah this is fine, I was just confused for a minute.